service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/C2DocumentBundle.java

```diff
@@ -50,8 +49,6 @@ public class C2DocumentBundle implements ApplicationsBundle {
     private final List<C2AdditionalOrdersRequested> c2AdditionalOrdersRequested;
     private final ParentalResponsibilityType parentalResponsibilityType;
     private final String applicantName;
-    private final String othersNotified;
-    private final List<Element<Other>> others;
```

Comment:
just want to check ... does this get stored on the case data anywhere or is it a temp field? just want to make sure removing doesn't break existing cases when they are loaded and deserialised.